src/getScore.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+const id = 'f4qNRrUKtWgEVFl8Y7jY';
+const urlPost = `${URL}${id}/scores/`;
+
+export default async function getScore() {
+  const data = await axios.get(urlPost);
+  return data;
```

Comment:
- [ ] Kindly change this to es6 arrow function syntax.

Reply:
This issue is well resolved! ✔️